samples/js/picasa/slurp/src/main.js

```diff
@@ -158,15 +161,15 @@ async function main(): Promise<void> {
   // necessary. Consider doing something about that.
 
   const j = (v: any) => jsonToNoms(v);
-  return db.commit(out, newStruct('', {
+  return db.commit(out, newStructWithType(makeStructType('', {value: valueType}), [newStruct('', {
     albums: j(albums),
     faces: j(faces),
     user: j(user),
-  }, {
-    meta: {
+  })]), {
```

Comment:
Not sure. If an optional argument is provided it needs to match. Maybe we declared meta as any?